imap/imapd.c

```diff
@@ -7708,25 +7708,24 @@ static void cmd_create(char *tag, char *name, struct dlist *extargs, int localon
 
                     // special use needs extended support, so pass through extargs
                     if (specialuse.len || uniqueid) {
+                        const char *sep = "";
+
                         prot_printf(s_conn->out, " (");
-                        int printsp = 0;
+
                         if (specialuse.len) {
-                            if (printsp) prot_putc(' ', s_conn->out);
-                            printsp = 1;
-                            prot_printf(s_conn->out, "USE (%s)", buf_cstring(&specialuse));
+                            prot_printf(s_conn->out, "%sUSE (%s)",
+                                        sep, buf_cstring(&specialuse));
```

Comment:
As `sep` here is the empty string, it can be removed completely from `prot_printf("%s", sep);`.

Reply:
Fixed